docs/painless/painless-contexts/painless-watcher-condition-context.asciidoc

```diff
@@ -1,38 +1,146 @@
 [[painless-watcher-condition-context]]
 === Watcher condition context
 
+include::painless-watcher-context-variables.asciidoc[]
+
+==== Watch condition
+
 Use a Painless script as a {xpack-ref}/condition-script.html[watcher condition]
-to test if a response is necessary.
+to test if a watch or a particular action should be executed. The condition
+will return a `boolean` value to determine if the condition was met. A
+condition can be used to short circuit an entire watch or within an action
+to determine execution of said action.
 
-*Variables*
+*Return*
 
-`params` (`Map`, read-only)::
-        User-defined parameters passed in as part of the query.
+`boolean`::
+        Expects `true` if the condition is met, and `false` otherwise.
 
-`ctx['watch_id']` (`String`, read-only)::
-        The id of the watch.
+*API*
 
-`ctx['execution_time']` (`ZonedDateTime`, read-only)::
-        The start time for the watch.
+The standard <<painless-api-reference, Painless API>> is available.
 
-`ctx['trigger']['scheduled_time']` (`ZonedDateTime`, read-only)::
-        The scheduled trigger time for the watch.
+The first example is a condition for the watch itself. This determines if any
+of the actions in the watch are executed. The example aggregates the total sold
```

Comment:
This may need to be re-worded as it sounds like you can select individual actions to execute for a watch based on a single condition.

Reply:
In other places, we use an "Example" label before the example intro (like Return and API). 

Rather than "first" and "second", I'd just use "the following" to introduce the examples.

*Example*

The following watch condition script controls overall execution of the watch based on the value of the seats sold for the plays in the data set. The script aggregates the total sold seats for each play and returns `true` if there is at least one play that has sold under $15,000 or over $50,000.